enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -119,7 +120,7 @@ func (r *runnerService) createAction(ctx context.Context, req *rnpb.RunRequest,
 		"--invocation_id=" + invocationID,
 		"--commit_sha=" + req.GetRepoState().GetCommitSha(),
 		"--target_branch=" + req.GetRepoState().GetBranch(),
-		// TODO(Maggie): Pass --interactive flag once all executors support the flag
+		"--interactive=" + fmt.Sprintf("%v", req.GetInteractive()),
```

Comment:
per the comment in the other PR wdyt about having the CI runner detect whether it's outputting to a terminal (and for workflows we can continue wrapping in a pty) and deciding whether to wrap in a pty that way?

fwiw this is pretty common, tools like bazel / git will detect whether stdout/stderr are not TTYs and if so they will disable fancy output.